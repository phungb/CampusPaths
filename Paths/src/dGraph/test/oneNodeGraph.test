# Tests if a node can be added to a graph

# Create a graph
CreateGraph graph1

# Add one node
AddNode graph1 n1

# Print the nodes in the graph
ListNodes graph1
ListChildren graph1 n1