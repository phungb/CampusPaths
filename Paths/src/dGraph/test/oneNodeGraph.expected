# Tests if a node can be added to a graph

# Create a graph
created graph graph1

# Add one node
added node n1 to graph1

# Print the nodes in the graph 
graph1 contains: n1
the children of n1 in graph1 are: